Fix test_multiplexed_metadata test in github actions

### Context

Currently trying to deploy an unrelated feature. However the `test_multiplexed_metadata` test is failing and preventing the deploy.

### Problem or idea

Tests pass locally but fail in github actions.

The failure message:

```bash
======================================================================
FAIL: test_multiplexed_metadata (scpca_portal.test.management.commands.test_load_data.TestLoadData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/unittest/mock.py", line 1379, in patched
    return func(*newargs, **newkeywargs)
  File "/home/user/code/scpca_portal/test/management/commands/test_load_data.py", line 222, in test_multiplexed_metadata
    self.assertEqual(sample_metadata_keys, expected_keys)
AssertionError: Lists differ: ['scp[403 chars]n', 'demux_method', 'demux_samples', 'filtered[152 chars]ade'] != ['scp[403 chars]n', 'cell_filtering_method', 'demux_method', '[255 chars]ade']

First differing element 25:
'demux_method'
'cell_filtering_method'

Second list contains 4 additional elements.
First extra element 36:
'sample_cell_estimates'

Diff is 857 characters long. Set self.maxDiff to None to see it.
```

### Solution or next step

Not really sure how to run this, going to try to run in a docker container that the github container uses.

--- a/.github/workflows/push.yml
+++ b/.github/workflows/push.yml
@@ -13,7 +13,6 @@ jobs:
           python-version: '3.10'
       - uses: pre-commit/action@v2.0.0
 
-
   run-linters:
     name: Run linters
     runs-on: ubuntu-latest
@@ -43,7 +42,7 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set perms
         run: chmod -R a+rw api
       - name: Build the stack

